core/db/migrate/20161012151358_opening_ledger_entries_for_store_credits.rb

```diff
@@ -0,0 +1,7 @@
+class OpeningLedgerEntriesForStoreCredits < ActiveRecord::Migration[5.0]
+  def change
+    Spree::StoreCredit.find_each do |store_credit|
+      Spree::StoreCreditLedgerEntry.generate_opening_ledger_entry_for(store_credit)
```

Comment:
Yes, wanted to have something that was easily testable though. Looking into options here.